allennlp/data/dataset_readers/penn_tree_bank.py

```diff
@@ -52,12 +57,15 @@ def _read(self, file_path):
         logger.info("Reading instances from lines in file at: %s", file_path)
 
         for parse in BracketParseCorpusReader(root=directory, fileids=[filename]).parsed_sents():
-            yield self.text_to_instance(parse.leaves(), [x[1] for x in parse.pos()], parse)
+            if self._use_pos_tags:
+                yield self.text_to_instance(parse.leaves(), [x[1] for x in parse.pos()], parse)
```

Comment:
Slight preference for assigning to a `pos_tags` variable in the if/else clause, and having just a single `yield`.  That way, if the `text_to_instance` method changes later, you only need to change it in one place.